Remotion/ObjectBinding/Web.Development.WebTesting.IntegrationTests/BocDateTimeValueControlObjectTest.cs

```diff
@@ -453,7 +453,7 @@ public void TestGetDateValidationError ()
 
       var validateErrors = bocDateTimeValue.GetDateValidationErrors();
 
-      Assert.That(validateErrors.Count, Is.EqualTo(1));
+      Assert.That(validateErrors.Count, Is.EqualTo(2));
       Assert.That(validateErrors.First(), Is.EqualTo("Enter a date."));
```

Comment:
Hmm...I just noticed this now, maybe the file's new?

We should replace both lines with 
```
Assert.That(validateErrors, Is.EquivalentTo(new [] { "Enter a date.", "second message." }));
```

That way we are not dependent on the order if validation errors. The current setup could have failed now that there's two errors.